sdk/storage/Azure.Storage.Files.DataLake/api/Azure.Storage.Files.DataLake.netstandard2.0.cs

```diff
@@ -360,6 +360,15 @@ public partial class DataLakeAccessPolicy
         [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
         public System.DateTimeOffset StartsOn { get { throw null; } set { } }
     }
+    public partial class DataLakeAclChangeFailedException : System.Exception, System.Runtime.Serialization.ISerializable
+    {
+        public DataLakeAclChangeFailedException(Azure.RequestFailedException exception, string continuationToken) { }
+        protected DataLakeAclChangeFailedException(System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context) { }
+        public string ContinuationToken { get { throw null; } }
+        public string ErrorCode { get { throw null; } }
+        public int Status { get { throw null; } }
+        public override void GetObjectData(System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context) { }
```

Comment:
We should only include a `ContinuationToken` property.  `Status` and `ErrorCode` can be retrieved from the `InnerException` if it was a `RequestFailedException`.  We should also override the base `Exception` .ctors so we can pass a string and inner exception: https://docs.microsoft.com/en-us/dotnet/api/system.exception.-ctor?view=netcore-3.1